app/controllers/api/v1/accounts_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'doorkeeper/grape/helpers'
+
+module API
+  module V1
+    class AccountsController < Grape::API
+      format :json
+
+      helpers Doorkeeper::Grape::Helpers
+
+      before do
+        doorkeeper_authorize!
+
+        def current_account
+          @current_account = Account.find(doorkeeper_token.resource_owner_id) if doorkeeper_token
+        end
+      end
+
+      resource :account do
```

Comment:
what is the meaning of resource ?

Reply:
@mod instead of this we can write ' get '/account' ' , but if we want more routes with account in future better to leave this